packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security.cy.ts

```diff
@@ -0,0 +1,90 @@
+describe("TESTS: Configuration => Subsystems => Elytron OIDC client", () => {
```

Comment:
I suppose this is not what we test here. Something like "TESTS: Access secured by Elytron OIDC client" would be more close to the subject of the testing in this spec.